libraries/os/OSAlloc.c

```diff
@@ -1,6 +1,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <dolphin.h>
+#include <string.h>
```

Comment:
I think you meant to include stdlib.h instead, since that's the header that contains malloc and free.